src/api/BoreholeGeometryExtensions.cs

```diff
@@ -1,42 +1,42 @@
 ﻿using BDMS.Models;
-using CsvHelper;
-using CsvHelper.Configuration;
-using Humanizer;
 using NetTopologySuite.Mathematics;
 using NetTopologySuite.Utilities;
-using System.Globalization;
 
-namespace BDMS.BoreholeGeometry;
+namespace BDMS;
 
-public static class Helper
+public static class BoreholeGeometryExtensions
 {
-    internal static readonly CsvConfiguration CsvConfig = new(new CultureInfo("de-CH"))
-    {
-        Delimiter = ";",
-        IgnoreReferences = true,
-        PrepareHeaderForMatch = args => args.Header.Humanize(LetterCasing.Title),
-        MissingFieldFound = null,
-    };
-
     /// <summary>
-    /// Get the CSV header <see cref="CsvHelper"/> expects to read a class <typeparamref name="T"/>.
-    /// Uses the map generated by <see cref="CsvHelper.CsvContext.AutoMap{T}()"/>.
-    /// If a property has multiple possible column names only the first is considered.
+    /// Get the TVD of <paramref name="depthMD"/> according to the <paramref name="geometry"/> if the geometry exists.
     /// </summary>
-    /// <typeparam name="T">The class to get the header for.</typeparam>
-    internal static string GetCSVHeader<T>()
+    /// <param name="geometry">The list of <see cref="BoreholeGeometryElement"/> representing the borehole's path geometry.</param>
+    /// <param name="depthMD">The measured depth (MD) at which to calculate the TVD.</param>
+    /// <returns>The TVD at <paramref name="depthMD"/> if the geometry exists; otherwise <see langword="null"/>.</returns>
+    internal static double? GetTVDIfGeometryExists(this List<BoreholeGeometryElement> geometry, double? depthMD)
     {
-        var context = new CsvContext(CsvConfig);
-        var map = context.AutoMap<T>();
-        return string.Join("; ", map.MemberMaps
-            .Select(m =>
+        if (geometry == null || geometry.Count < 2)
+        {
+            if (depthMD != null && depthMD >= 0)
             {
-                var name = m.Data.Names.FirstOrDefault(m.Data.Member.Name);
-                return m.Data.IsOptional ? $"[{name}]" : name;
-            }));
-    }
+                // Return the depthMD unchanged as if the borehole is perfectly vertical and infinitely long.
+                return depthMD;
+            }
+        }
+        else if (depthMD != null)
+        {
+            try
+            {
+                // Assuming GetDepthTVD is another extension method or a suitable method of List<BoreholeGeometryElement>
+                return geometry.GetDepthTVD(depthMD.Value);
```

Comment:
Q: Könnte die Mehtode `GetDepthTVD` nicht `null` returnen anstatt eine Exception zu schmeissen? So wie ich das sehe, machen wir nie etwas mit der `ArgumentOutOfRangeException`.

Reply:
Einige Tests erwarten die Exception, ich mache daher das Fass jetzt nicht auf ;) Ich weiss nicht genau was die Überlegung dahinter war, vielleicht damit beim Testen besser zwischen Argument out of range und keinem Input unterschieden werden kann.

Reply:
Okay. Sorry, habe mich verschaut. Dachte der Endpoint sei neu.